bertopic/_bertopic.py

```diff
@@ -73,12 +72,23 @@
 )
 import bertopic._save_utils as save_utils
 
-# Visualization
-import plotly.graph_objects as go
 
 logger = MyLogger()
 logger.configure("WARNING")
 
+try:
+    from bertopic import plotting
+    import plotly.graph_objects as go
+
+except ModuleNotFoundError as e:
+    if "No module named 'plotly'" in str(e):
```

Comment:
I'm wondering whether this is a good enough check to see whether plotly is installed. Checking the string might be unstable. Perhaps use `importlib` instead?

```python
plotly_available = importlib.util.find_spec("plotly") is not None
```